### PR TITLE
add usdc and osmo

### PR DESCRIPTION
**Pull request by wanesurf**
This change is [Reviewable](https://reviewable.io/reviews/CoreumFoundation/token-registry/6)

### CODE REVIEW

File: mainnet/assets.json [hunk 1]
```diff
@@ -23,6 +23,58 @@
...
+        },
+        {
+            "denom": "ibc/E1E3674A0E4E1EF9C69646F9AF8D9497173821826074622D831BAB73CCB99A2D",
+            "description": "USDC from Noble",
+            "ibc_info": {
+
+            },
+            "logo_URIs": {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
```
Comment (akhlopiachyi) [inline mainnet/assets.json:35]: images should be inside of this repo, not somewhere else

File: mainnet/assets.json [hunk 2]
```diff
@@ -23,6 +23,58 @@
...
+            },
+            "urls": {
+                "website": "https://nobleassets.xyz",
+                "github": "https://github.com/noble-assets/noble",
+                "whitepaper": "#"
```
Comment (akhlopiachyi) [inline mainnet/assets.json:40]: if there is not whitepaper link, don't set it here

File: mainnet/assets.json [hunk 3]
```diff
@@ -23,6 +23,58 @@
...
+            },
+            "social_media": {
+                "linkedin": "#",
+                "twitter": "https://twitter.com/noble_xyz",
+                "instagram": "#",
+                "facebook": "#",
+                "discord": "#",
+                "youtube": "#",
+                "telegram": "#",
+                "tiktok": "#"
```
Comment (akhlopiachyi) [inline mainnet/assets.json:50]: if there is not links, don't set them here to #

File: mainnet/assets.json [hunk 4]
```diff
@@ -23,6 +23,58 @@
...
+        },
+        {
+            "denom": "ibc/13B2C536BB057AC79D5616B8EA1B9540EC1F2170718CAFF6F0083C966FFFED0B",
+            "description": "Osmo from Osmosis",
+            "ibc_info": {
+
+            },
+            "logo_URIs": {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.png",
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/osmo.svg"
```
Comment (akhlopiachyi) [inline mainnet/assets.json:61]: the same as above

File: mainnet/assets.json [hunk 5]
```diff
@@ -23,6 +23,58 @@
...
+            },
+            "social_media": {
+                "linkedin": "#",
+                "twitter": "https://twitter.com/osmosiszone",
+                "instagram": "#",
+                "facebook": "#",
+                "discord": "https://discord.com/invite/osmosis",
+                "youtube": "#",
+                "telegram": "https://t.me/osmosis_chat",
+                "tiktok": "#"
```
Comment (akhlopiachyi) [inline mainnet/assets.json:76]: the same as above

**Review by akhlopiachyi — CHANGES_REQUESTED** [review]

**Review by wanesurf — COMMENTED** [review]
> Reviewed 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/token-registry/6)* status: all files reviewed, 5 unresolved discussions (waiting on @akhlopiachyi, @dzmitryhil, @miladz68, @wojtek-coreum, and @ysv)

___
*[`mainnet/assets.json` line 35 at r1](https://reviewable.io/reviews/CoreumFoundation/token-registry/6#gh-1452540697:-NoE7xFv366QX9nENSP7:b-896fix) ([raw file](https://github.com/CoreumFoundation/token-registry/blob/2c9ff9ffcb9ab11c9648bef280012fe149f63e7c/mainnet/assets.json#L35)):*
<details><summary><i>Previously, akhlopiachyi (Alex Khlopiachyi) wrote…</i></summary><blockquote>

images should be inside of this repo, not somewhere else
</blockquote></details>

Done.
___
*[`mainnet/assets.json` line 40 at r1](https://reviewable.io/reviews/CoreumFoundation/token-registry/6#gh-1452541285:-NoE7xfo33-DZVgtsZUj:b-896fix) ([raw file](https://github.com/CoreumFoundation/token-registry/blob/2c9ff9ffcb9ab11c9648bef280012fe149f63e7c/mainnet/assets.json#L40)):*
<details><summary><i>Previously, akhlopiachyi (Alex Khlopiachyi) wrote…</i></summary><blockquote>

if there is not whitepaper link, don't set it here
</blockquote></details>

Done.
___
*[`mainnet/assets.json` line 50 at r1](https://reviewable.io/reviews/CoreumFoundation/token-registry/6#gh-1452541827:-NoE7y1f61POGqCPi5-r:b-896fix) ([raw file](https://github.com/CoreumFoundation/token-registry/blob/2c9ff9ffcb9ab11c9648bef280012fe149f63e7c/mainnet/assets.json#L50)):*
<details><summary><i>Previously, akhlopiachyi (Alex Khlopiachyi) wrote…</i></summary><blockquote>

if there is not links, don't set them here to #
</blockquote></details>

Done.
___
*[`mainnet/assets.json` line 61 at r1](https://reviewable.io/reviews/CoreumFoundation/token-registry/6#gh-1452542091:-NoE7ySE06axJDr4X5UQ:b-896fix) ([raw file](https://github.com/CoreumFoundation/token-registry/blob/2c9ff9ffcb9ab11c9648bef280012fe149f63e7c/mainnet/assets.json#L61)):*
<details><summary><i>Previously, akhlopiachyi (Alex Khlopiachyi) wrote…</i></summary><blockquote>

the same as above
</blockquote></details>

Done.
___
*[`mainnet/assets.json` line 76 at r1](https://reviewable.io/reviews/CoreumFoundation/token-registry/6#gh-1452542233:-NoE7ytm-Xpvzmag2THK:b-896fix) ([raw file](https://github.com/CoreumFoundation/token-registry/blob/2c9ff9ffcb9ab11c9648bef280012fe149f63e7c/mainnet/assets.json#L76)):*
<details><summary><i>Previously, akhlopiachyi (Alex Khlopiachyi) wrote…</i></summary><blockquote>

the same as above
</blockquote></details>

Done.

**Review by akhlopiachyi — COMMENTED** [review]
> Reviewed 5 of 5 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/CoreumFoundation/token-registry/6)* status: all files reviewed, 5 unresolved discussions (waiting on @wojtek-coreum and @ysv)

**Pull request merged**